mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule Fields.MixProject do
     [
       app: :fields,
       version: "0.1.0",
```

Comment:
consider bumping the version in the `mix.exs` each time an update is made. e.g: `0.1.0` >> `0.1.1`